test/resources/frameworks/qunit/qunit.css

```diff
@@ -0,0 +1,305 @@
+/*!
```

Comment:
We’ll want to remove this file and replace references to it with `node_modules/qunitjs/qunit/qunit.css`.